### PR TITLE
Link to gRPC samples

### PR DESCRIPTION
**Pull request by patriknw**
Not as nice as the downloadable zip files we have in the guide, but good enough for now.

### CODE REVIEW

File: docs/src/main/paradox/grpc.md [hunk 1]
```diff
@@ -97,6 +97,20 @@ Java
 This example includes an application specific `ShoppingCartService`, which is unrelated to Akka Projections gRPC,
 but it illustrates how to combine the `EventProducer` service with other gRPC services.
 
+## Sample projects
+
+Source code and build files for complete sample projects can be found in `akka/akka-projection` GitHub repository. 
+
+Scala:
```
Comment (patriknw) [inline docs/src/main/paradox/grpc.md:104]: btw, I tried to place this in Scala/Java tabs, but couldn't find out the syntax for non-snip in tabs.

Comment (johanandren) [reply docs/src/main/paradox/grpc.md:104]: Should be the same as in Akka docs, right? 

```
@@@ div { .group-scala }
 
Bla bla bla

@@@

**Review by johanandren — APPROVED** [review]

File: docs/src/main/paradox/grpc.md (hunk 1 above)
Comment (patriknw) [reply docs/src/main/paradox/grpc.md:104]: That works, but not very clear that there are Java and Scala samples to select from. Then you have to select the language first via some of the other tabs.

Comment (johanandren) [reply docs/src/main/paradox/grpc.md:104]: Ah, you don't get the actual tabs, didn't think of that.

**Pull request merged**